main.cpp

```diff
@@ -41,22 +41,458 @@ If you need inspiration for what to write, take a look at previously approved st
 /*
  UDT 1:
  */
+#include <iostream>
+#include <string>
+#include <cmath>
+
+using namespace std;
```

Comment:
**never write this.**  always spell out the namespace of anything in the STL when naming objects or functions from it.
- [ ] revise